dedicated-examples/extended-error-tracking/README.MD

```diff
@@ -0,0 +1,28 @@
+# Log Client-Side Errors
+
+When a JavaScript error occurs on a tested web page, TestCafe stops test execution and posts an error message. Requests to foreign resources (e.g. third-party APIs) might cause unexpected errors that are not relevant to your test. In that case, you might want to filter those errors out to ignore them during the test.
+
+This example demonstrates how to expand on TestCafe's standard error logging and filter client-side errors that happen during a test.
```

Comment:
`This example demonstrates how to expand`. `extend` is a better word for this place.

Reply:
> Third-party code execution (e.g. logging and analytics services, controls) might cause unexpected errors that are not relevant to the tested app. You might want to filter those errors out to ignore them during the test.